src/main/java/duke/handlers/HandlerFactory.java

```diff
@@ -0,0 +1,73 @@
+package duke.handlers;
+
+import duke.enums.Command;
+import duke.exceptions.DukeException;
+import duke.service.Service;
+
+
+public class HandlerFactory {
+    String command;
+    String taskName;
+    String flag;
+    String flagOption;

```

Comment:
Since you created setter functions for these, these variables should be `private`